Dockerfile

```diff
@@ -23,6 +24,7 @@ RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -a -o manager main.go
 FROM gcr.io/distroless/static:nonroot
 WORKDIR /
 COPY --from=builder /workspace/manager .
+COPY --from=builder /workspace/manifests/ ./manifests/
```

Comment:
Instead of adding the manifests to the container image, why not embed them in the Go binary using [go embed](https://pkg.go.dev/embed). I believe this would simplify some of the FS setup and file load?

Reply:
I'm not sure I follow why it's more difficult. The current approach seems to involve more steps than using `embed`, since you need to include the COPY steps in the Dockerfile. 
Also things like `"manifests/base"` are hardcoded so how it different than embed?

Am I missing something?

Reply:
yesss